### ISSUE TITLE
Get 'Segmentation Fault' while running benchmark for NEON

### ISSUE THREAD

**Issue by mstfc** [ARM-software/ComputeLibrary#245]
Hi all, 

I get the 'Segmentation Fault' when running ACL benchmark for NEON.
ACL benchmark for CL can run to the end successfully.
I just follow the instructions that ACL document mentioned.
Dose anyone hit the same issue? 
I ran the benchmark on V17.09 and V17.10 and both got the same error.

Thanks for your help.

**Comment by GeorgeARM** [comment]
Hello @mstfc,

Can you provide more information?
How do you build the library? Where it fails (Any particular tests)? Where do you run it?

**Comment by mstfc** [comment]
Hi George,

Thanks for reply.
I ran it on Raspberry PI3 model B.
Cross-compiled ACL on Linux X86_64 PC, tool chain version is (Linaro GCC 4.9-2017.01)4.9.4.
Here is the compile command : 
 scons Werror=1 -j4 debug=1 neon=1 cppthreads=0 openmp=0 opencl=0 os=linux arch=armv7a standalone=0 examples=1 embed_kernels=1 benchmark_tests=1

I traced the code and found it failed at AlexNetNetwork.h : run() : act2.run();
It's just the first test case.

I also found that, in AlexNet.cpp : NEAlexNetFixture, there are 2 NEConvolutionLayer and is different from CL. 
I replaced one NEConvolutionLayer with NEDirectConvolutionLayer just like CL does and the first can finish AlexNetNetwork.h : run() function but got " *** Error in `./arm_compute_benchmark': double free or corruption (out): 0x7667b008 *** " error message after than.

I'm still trying to debug, but the progress is too slow.
Hope that I can get some hints from you.
Thank you.

Best regards,
Henry
